### ISSUE TITLE
Support CMYK JPEG files

### ISSUE THREAD

**Issue by foresmac** [vokal/vip#107]
Go 1.5 now [supports them](https://golang.org/pkg/image/color/#CMYK); may just require a recompile on 1.5. Add test case with CMYK JPEG file.

**Comment by scottferg** [comment]
Yup. I'd be curious what a test would look like.

**Comment by foresmac** [comment]
Steve from Trunq posted some test JPEGs which he couldn't upload because they were saved as CMYK by a graphic designer.

**Comment by jrit** [comment]
Part of the problem is that we shouldn't be serving CMYK jpegs either, so if they are accepted for upload, they should always be converted to RGB before they are stored on S3 even for serving the full original size.

**Comment by foresmac** [comment]
Yeah, that's pretty trivial to do and what I had always envisioned.
